.pnp.cjs

```diff
@@ -53,7 +53,7 @@ const RAW_RUNTIME_STATE =
     ]],\
     ["@aashutoshrathi/word-wrap", [\
       ["npm:1.2.6", {\
-        "packageLocation": "../../.yarn/berry/cache/@aashutoshrathi-word-wrap-npm-1.2.6-5b1d95e487-10c0.zip/node_modules/@aashutoshrathi/word-wrap/",\
+        "packageLocation": "../../../.yarn/berry/cache/@aashutoshrathi-word-wrap-npm-1.2.6-5b1d95e487-10c0.zip/node_modules/@aashutoshrathi/word-wrap/",\
```

Comment:
Can we remove now `.pnp.cjs` and `.yarn/install-state.gz` from PR? 
Seems like a noise

Reply:
Done